# Statistical Analysis of High-Throughput Genomic and Transcriptomic Data
Fall/Herbst-semester 2022

## Lectures
Mondays 9.00-9.45 (Y27-H-46), 10.00-10.45 (Y27-H-46)

## Exercises
Monday 11.00-11.45 (Y01-F-50)

## Lecturers

[Dr. Hubert Rehrauer](http://www.fgcz.ch/the-center/people/rehrauer.html), Group Leader of Genome Informatics at [FGCZ](http://www.fgcz.ch/)  

[Prof. Dr. Mark Robinson](https://robinsonlabuzh.github.io/), Associate Professor of Statistical Genomics, [DMLS](https://www.mls.uzh.ch/en.html), UZH  

## Schedule

| Date  | Lecturer | Topic | Exercise | JC1 | JC2 |
| --- | --- | --- | --- | --- | --- |
| 19.09.2022  | Mark + Hubert  | admin; mol. bio. basics | quarto; git(hub) | | |
| 26.09.2022  | Mark | interactive technology/statistics session  | group exercise: technology pull request | | |
| 03.10.2022  | Hubert | NGS intro; exploratory data analysis | EDA in R | | |
| 10.10.2022  | Mark | limma + friends | linear model simulation + design matrices | | |
| 17.10.2022  | Hubert | mapping  | Rsubread | | |
| 24.10.2022  | Hubert | RNA-seq quantification | RSEM  | [Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0030161) JK DW | X |
| 31.10.2022  | Mark | edgeR+friends 1 | basic edgeR/voom | [Cell2location maps fine-grained cell types in spatial transcriptomics](https://www.nature.com/articles/s41587-021-01139-4) (JS, GM)  | X |
| 07.11.2022  | Mark | edgeR+friends 2 | advanced edgeR/voom | X | X |
| 14.11.2022  | YYY | hands-on session #1: RNA-seq  | FASTQC/Salmon/etc. | X | X |
| 21.11.2022  | Hubert | single-cell 1: preprocessing, dim. reduction, clustering | clustering | X | [Comparison of Transformations for Single-Cell RNA-Seq Data](https://www.biorxiv.org/content/10.1101/2021.06.24.449781v3.full.pdf) (GA, RE)|
| 28.11.2022 | YYY | hands-on session #2: cytometry | cytof null comparison | [Integrated analysis of multimodal single-cell data](https://www.sciencedirect.com/science/article/pii/S0092867421005833) (ME, FR) | X |
| 05.12.2022  | Mark | single-cell 2: clustering, marker gene DE  | marker gene DE | [Differential abundance testing on single-cell data using k-nearest neighbor graphs](https://www.nature.com/articles/s41587-021-01033-z) (DK) | X
| 12.12.2022  | YYY | hands-on session #3: single-cell RNA-seq (cell type definition, differential state)  | full scRNA-seq pipeline | X | X |
| 19.12.2022  | Mark | loose ends: HMM, EM, robustness   | segmentation, peak finding | X | X |
 

## Course material

Assuming you have git installed locally, you can check out the entire set of course materials with the following command (from command line):
```
git clone https://github.com/sta426hs2022/material.git
```  
and get updates by running `git pull` at any later time in the same directory.

Alternatively, to retrieve a ZIP file of the repository, you can click on the (green) 'Code' button (top right of main panel) and then click 'Download ZIP'.
