# Chromatin immune precipitation sequencing (ChIP-seq)

## Overview

ChIP-seq, which combines chromatin immunoprecipitation (ChIP) assays with high-throughput sequencing, is a method to identify DNA-binding sites, primarily for transcription factors and other DNA-binding proteins.

Target DNA regions in the genome for a target protein is coprecipated in ChIP protocol with a specify antibody. Purfied and amplified, the bound DNA fragments are the subject to high-throughput sequencing.

The short sequenced fragments are mapped to a reference genome of choice via alignment. The regions of enriched read counts (tags) are called/identified by different peak calling algorithms.

## _Link_

<!-- ChIP-seq -> DNA-binding sites discovery -> Poisson Model/Negative Binomial Model -->

| Technology    | Application   | Statistics |
| ------------- | ------------- | ---------- |
|  ChIP-seq | Analysis/discovery of DNA binding sites (regulatory elements), chromatin interaction identification | Negative binomial/dynamic Poisson distribution |

## Links and references

Thomas, R., Thomas, S., Holloway, A. K., & Pollard, K. S. (2017). Features that define the best ChIP-seq peak calling algorithms. _Briefings in bioinformatics_, 18(3), 441â€“450. https://doi.org/10.1093/bib/bbw035

Zhang, Y., Liu, T., Meyer, C.A. et al. Model-based Analysis of ChIP-Seq (MACS). _Genome Biol_ 9, R137 (2008). https://doi.org/10.1186/gb-2008-9-9-r137

## Group Members

[daggermaster3000](https://github.com/daggermaster3000), [wyq977](https://github.com/wyq977), [sarloet](https://github.com/sarloet)
