---
title: "Ex2B"
author: "Metry Célia, Ylva Carlen, Elina Jansone"
date: '2022-09-26'
output: html_document
---

# ATAC- Sequencing

## What is ATAC- Sequencing and how does it work?

ATAC-Sequencing is short for Assay for Transposase-Accessible Chromatin with high-throughput sequencing. It uses a hyperactive transposase Tn5 and relies on NGS for library construction. This method can be used to construct a genome-wide chromatin accessability map and can be used to compare epigenetic landscapes of different cell types. The principle is based on the fact that chromation accessability depends on the gene region. It can be more or less accessible depending on histone density and modifications. Using Tn5 for tagmentation it mainly binds on accessible part of the DNA. After cell preparation and fragmentation, the cut ends are inserted with sequencing adaptors.Those parts are then amplified and this results in the ATAC-Seq Library by using barcoded Primers. Data Analysis is performed and accessible chromatin regions are shown as peaks.

------------------------------------------------------------------------

## Technology, Application, Statistics

#### Technology:

ATAC-Sequencing

#### Application:

-   Genome-wide chromatin accessibility landscape

-   Nucleasmoe mapping

-   Transcritpion binding side mapping

-   DNA methylation side mapping

#### Statisitcs:

All information for ATAC-seq data processing, including data quality control, genome alignment, peak analysis, clustering and TF motif analysis, are described on [this website](https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/algorithms/overview).

With the results of the peak analysis a **Gene set enrichment analysis (GSEA)** can be performed. This is a method for identifying gene or protein classes that are over represented in a large group of genes or proteins.

To identify the different chromatin accessibility the enrichment of motifs in the peaks can be analysed. With the ATAC-seq the binding of TF can be investigated, which are called TF motif. The differential TF motif between two groups can be calculated using a **t-Test**.

------------------------------------------------------------------------

## Sources

NCBI[\^1]

ActiveMotif[\^2]

springer[\^3]

[\^1]: <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4374986/>\
[\^2]: <https://www.activemotif.com/blog-atac-seq>\
[\^3]: <https://link.springer.com/content/pdf/10.1007/s10067-022-06310-z.pdf> 
